packages/dart/noports_core/lib/src/sshnp/impl/sshnp_unsigned_impl.dart

```diff
@@ -87,7 +90,7 @@ class SshnpUnsignedImpl extends SshnpCore
         ..sharedBy = params.clientAtSign
         ..sharedWith = params.sshnpdAtSign
         ..metadata = (Metadata()..ttl = 10000),
-      '$localPort ${srvdChannel.port} ${keyUtil.username} ${srvdChannel.host} $sessionId',
+      '$localPort ${srvdChannel.clientPort} ${keyUtil.username} ${srvdChannel.host} $sessionId',
```

Comment:
again, some renaming of variables in srvdChannel so they make more sense (better readability)